test_runner/src/main/kotlin/ftl/args/ValidateIosArgs.kt

```diff
@@ -15,6 +15,13 @@ fun IosArgs.validate() = apply {
     checkResultsDirUnique()
     assertAdditionalIpas()
     validType()
+    assertGameloop()
+}
+
+fun IosArgs.assertGameloop() {
```

Comment:
you could check also if `scenariosNumber.size` is below `1024`

Reply:
👍